netstandard/ref/System.IO.Pipes.cs

```diff
@@ -78,6 +82,7 @@ public enum PipeDirection
     public enum PipeOptions
     {
         Asynchronous = 1073741824,
+        CurrentUserOnly = 536870912,
```

Comment:
Cool, this addition is what is needed for https://github.com/dotnet/corefx/issues/30170

Reply:
A related thing is that System.IO.Pipes.AccessControl stuff is listed as ".NET Standard + Platform Extensions" but since it can't do anything useful we should remove the package from the whole extensions.

Reply:
Package issue being tracked at https://github.com/dotnet/corefx/issues/31190

Reply:
Excited to see this 😄 we need this for [PowerShell Editor Services](https://github.com/powershell/powershelleditorservices)

Reply:
Is there any way to take advantage of this before it's released? Like by passing in the int value?

Trying to get rid of a mess of pinvoking and shelling out to chmod for a cross plat solution.

Reply:
@tylerl0706 it depends on where you are trying to run. If you are trying to run on .NET Core 2.1 then yes you can target your library to .NET Core 2.1 to get this API (if you are targeting .NET Standard you could probably cast the value but that will only work if you run that library on a platform that supports it).

Reply:
Yeah we are targeting .NET Standard (and need to) because we also run on .NET Framework.

That said... our chmoding could be removed if we can cast the value, do you think you could elaborate on that?

Reply:
So https://apisof.net/catalog/System.IO.Pipes.PipeOptions.CurrentUserOnly is only supported on .NET Core 2.1+ and so if you try to do the cast the code will fail if you try to run it anywhere but .NET Core. However if you still wish to try and do it you would just cast the integer value to PipeOptions like such `(PipeOptions)536870912` and pass it along to an existing API that takes PipeOptions, just be warned it will fail anywhere but .NET Core 2.1+

Reply:
Thanks for the info @weshaggard. We will be dropping support of .NET Core 2.0 in ~6mo so that's something we can consider then.

Unless this PR gets merged in, then I'm guessing this will work cross plat and cross runtime?

Reply:
It works cross platform in .NET Core 2.1+ but currently not cross runtime as .NET Core is the only runtime that supports it currently.

Reply:
When this gets pulled into standard, will in work cross-runtime? Or no?

Reply:
Yes for any runtime versions that claim to support the new standard. 

**dislaimer** not all runtimes support all APIs even if they clam to support a new standard but that is general goal.